packages/http-client/src/ceramic-http-client.ts

```diff
@@ -99,16 +101,18 @@ export class CeramicClient implements CeramicApi {
     }
   }
 
-  get did(): DID | undefined {
+  get didVerifier(): DidVerifier | undefined {
+    return this.context.didVerifier
+  }
+
+  get did(): ThreadedDid | undefined {
```

Comment:
I don't think we want to change the http client like this.  This has implications for app devs and is a breaking change to our API.  It also now requires app developers to provide a ThreadedDid just to sign the messages for creating commits, which is probably overkill added complexity they don't need.

I think if we need a ThreadedDid in the `Ceramic` object in the `core` package, we need to put it in a place that isn't part of the shared contract with the http client

Reply:
We could change the http client to not inherit from Ceramic, and instead just have it as a dependency.

Reply:
I'm definitely interested in breaking up the http client from core and allowing them to diverge more.

But I don't think we need to do that just to deliver this feature?  We could just take the `did` field out of the common interface and let each implementation define it how it needs.